remote_monitoring_demo/remote_monitoring_demo.sh

```diff
@@ -40,7 +40,6 @@ function usage() {
 	echo "c = supports minikube and openshift cluster-type"
 	echo "d = duration of benchmark warmup/measurement cycles"
 	echo "p = expose prometheus port"
-	echo "u = Kruize UI Image. Default - quay.io/kruize/kruize-ui:<version as in package.json>"
```

Comment:
Need to keep this line.
echo "u = Kruize UI Image. Default - quay.io/kruize/kruize-ui:<version as in package.json>"

Reply:
done